bin/dev

```diff
@@ -2,6 +2,16 @@
 require 'open3'
 require 'pathname'
 require 'rbconfig'
+require 'tempfile'
+
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'docker-api', '~> 2.0'
+  gem 'outstand-tty-command', '~> 0.10'
```

Comment:
I forked `tty-command` to get a release that wouldn't spam warnings on ruby 2.7.